apps/web/src/components/NavMenu/MainNavMenu.tsx

```diff
@@ -167,47 +181,29 @@ const MainNavMenu = () => {
                   description: "Label for the main navigation",
                 })}
                 data-state={isMenuOpen ? "open" : "closed"}
-                data-h2-background-color="base(foreground) l-tablet:all(black.9)"
-                data-h2-radius="base(rounded) l-tablet(initial)"
-                data-h2-padding="base(1px 0) l-tablet(x1 0)"
+                className="rounded-md bg-white py-6 sm:rounded-none sm:bg-gray-700 sm:py-6 dark:bg-gray-600 sm:dark:bg-gray-700"
               >
-                <div
-                  data-h2-wrapper="l-tablet(center, large, x2)"
-                  data-h2-display="l-tablet(flex)"
-                  data-h2-justify-content="l-tablet(space-between)"
-                  data-h2-align-items="base(center)"
+                <Container
+                  center
+                  size={{ sm: "lg" }}
+                  className="items-center px-0 sm:flex sm:justify-between sm:px-12"
```

Comment:
If you change `sm:px-6` to `sm:px-12` it will work

Reply:
Fixed in [adjust padding](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13799/commits/165749e126b2ccca5d66cecb0be0a5b9df308c69)